contracts/PaperManager.sol

```diff
@@ -26,4 +26,10 @@ contract PaperManager {
     function addFeedBack(string memory _ipfsHash, string memory _feedback) public {
         papers[_ipfsHash].addFeedback(_feedback);
     }
+
+    function primaryPaperChecking(bool _checks, string memory _ipfsHash) public {
+        if (_checks) {
+            papers[_ipfsHash].primaryChecking();
+        }
+    }
```

Comment:
What is this function supposed to do ? Is it because the `PaperManager` is the owner of the `Paper`, not the editor ? To save cost, maybe the editor should be able to call the functions directly on the `Paper` contract, or they should be in the `PaperManager` in the first place. Wrapping function sadly adds unnecessary operations which cost gas.